2023-03-25 13:49:47,592 - INFO - [MainThread] - Initialized router P7 at port 5007 PID 23680
2023-03-25 13:49:47,595 - INFO - [MainThread] - Adding a connection to Vulnerar
2023-03-25 13:49:47,595 - INFO - [Connect-Vulnerar] - Calling connect_to_rendezvous_point
2023-03-25 13:49:47,596 - INFO - [Connect-Vulnerar] - Connect to rendezvous point Vulnerar
2023-03-25 13:49:47,599 - INFO - [MainThread] - Adding listener for RMK
2023-03-25 13:49:47,599 - INFO - [Establish-RMK] - Establishing RP to RMK
2023-03-25 13:49:47,603 - INFO - [Connect-Vulnerar] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:49:47,614 - INFO - [Establish-RMK] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:49:47,633 - INFO - [MainThread] - Starting testing thread
2023-03-25 13:49:47,640 - INFO - [TESTER] - Started tester
2023-03-25 13:49:47,758 - INFO - [MainThread] - Starting connection thread
2023-03-25 13:49:47,785 - INFO - [MainThread] - Starting log thread
2023-03-25 13:49:48,275 - INFO - [Connect-Vulnerar] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:49:48,359 - INFO - [Establish-RMK] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:49:48,360 - INFO - [Establish-RMK] - Connecting to guard node 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13)... (None)
2023-03-25 13:49:48,566 - INFO - [Connect-Vulnerar] - Connected to rendezvous point Vulnerar
2023-03-25 13:49:48,566 - INFO - [Connect-Vulnerar] - Waiting for peer to open stream
2023-03-25 13:49:48,620 - INFO - [Connect-Vulnerar] - Received cell: <class 'torpy.cells.CellDestroy'>
2023-03-25 13:49:48,621 - WARNING - [Connect-Vulnerar] - Could not connect to rendezvous point Vulnerar: Receive CellDestroy from RP, 11 attempts left...
2023-03-25 13:49:49,120 - INFO - [Establish-RMK] - Creating new circuit #80000001 with 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13) router...
2023-03-25 13:49:49,307 - INFO - [Establish-RMK] - Building -1 hops circuit...
2023-03-25 13:49:49,308 - INFO - [Establish-RMK] - Extending the circuit #80000001 with 46.4.101.213:9000 (RMK; Tor 0.4.8.0-alpha-dev)...
2023-03-25 13:49:49,308 - INFO - [Establish-RMK] - Getting descriptor for 46.4.101.213:9000 (RMK; Tor 0.4.8.0-alpha-dev)...
2023-03-25 13:49:49,313 - INFO - [Establish-RMK] - Connecting to guard node 213.232.235.83:42069 (back2bsd; Tor 0.4.7.13)... (Internal dir client)
2023-03-25 13:49:49,593 - INFO - [Establish-RMK] - Creating new circuit #80000002 with 213.232.235.83:42069 (back2bsd; Tor 0.4.7.13) router...
2023-03-25 13:49:49,666 - INFO - [Establish-RMK] - Building 0 hops circuit...
2023-03-25 13:49:49,666 - INFO - [Establish-RMK] - Stream #1: creating attached to #80000002 circuit...
2023-03-25 13:49:49,666 - INFO - [Establish-RMK] - Stream #1: connecting to hsdir
2023-03-25 13:49:49,735 - INFO - [RecvLoop_213.232] - Stream #1: connected (remote ip '')
2023-03-25 13:49:50,010 - INFO - [RecvLoop_213.232] - Stream #1: remote disconnected (reason = DONE)
2023-03-25 13:49:50,010 - INFO - [Establish-RMK] - Stream #1: closing (state = Disconnected)...
2023-03-25 13:49:50,010 - INFO - [Establish-RMK] - Got descriptor
2023-03-25 13:49:50,374 - INFO - [Establish-RMK] - <class 'torpy.circuit.TorCircuit'>
2023-03-25 13:49:50,738 - INFO - [Establish-RMK] - Rendezvous established (CellRelayRendezvousEstablished())
2023-03-25 13:49:50,738 - INFO - [Establish-RMK] - Waiting for connections at relay RMK for cookie b'eb688e4f52df90278061' ...
2023-03-25 13:49:52,887 - INFO - [Establish-RMK] - Got REND2 message from RMK
2023-03-25 13:49:52,887 - INFO - [Establish-RMK] - Entered circuit context with id 0x80000001
2023-03-25 13:49:52,887 - INFO - [Establish-RMK] - Stream #2: creating attached to #80000001 circuit...
2023-03-25 13:49:52,887 - INFO - [Establish-RMK] - Stream #2: connecting to ('127.0.0.1', 5000)
2023-03-25 13:49:53,300 - INFO - [RecvLoop_97.113.] - Stream #2: connected (remote ip '127.0.0.1')
2023-03-25 13:49:53,492 - INFO - [Establish-RMK] - Sending HELLO to peer
2023-03-25 13:49:53,492 - INFO - [Establish-RMK] - Sending HELLO to peer: b'\x80\x04\x95&\x00\x00\x00\x00\x00\x00\x00]\x94}\x94(\x8c\x07Message\x94\x8c\x05HELLO\x94\x8c\x04Peer\x94\x8c\x02P7\x94ua.'
2023-03-25 13:49:53,493 - INFO - [RECEIVER] - Added neighbour P5
2023-03-25 13:49:53,631 - INFO - [Connect-Vulnerar] - Calling connect_to_rendezvous_point
2023-03-25 13:49:53,631 - INFO - [Connect-Vulnerar] - Connect to rendezvous point Vulnerar
2023-03-25 13:49:53,634 - INFO - [Connect-Vulnerar] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:49:53,834 - INFO - [Connect-Vulnerar] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:49:54,094 - INFO - [Connect-Vulnerar] - Connected to rendezvous point Vulnerar
2023-03-25 13:49:54,094 - INFO - [Connect-Vulnerar] - Waiting for peer to open stream
2023-03-25 13:49:54,145 - INFO - [Connect-Vulnerar] - Received cell: <class 'torpy.cells.CellDestroy'>
2023-03-25 13:49:54,145 - WARNING - [Connect-Vulnerar] - Could not connect to rendezvous point Vulnerar: Receive CellDestroy from RP, 10 attempts left...
2023-03-25 13:49:59,155 - INFO - [Connect-Vulnerar] - Calling connect_to_rendezvous_point
2023-03-25 13:49:59,155 - INFO - [Connect-Vulnerar] - Connect to rendezvous point Vulnerar
2023-03-25 13:49:59,157 - INFO - [Connect-Vulnerar] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:49:59,354 - INFO - [Connect-Vulnerar] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:49:59,620 - INFO - [Connect-Vulnerar] - Connected to rendezvous point Vulnerar
2023-03-25 13:49:59,620 - INFO - [Connect-Vulnerar] - Waiting for peer to open stream
2023-03-25 13:50:00,108 - INFO - [Connect-Vulnerar] - Received cell: <class 'torpy.cells.CellRelay'>
2023-03-25 13:50:00,108 - INFO - [Connect-Vulnerar] - Stream opened successfully
2023-03-25 13:50:00,108 - INFO - [Connect-Vulnerar] - Sending HELLO to peer: b'\x80\x04\x95&\x00\x00\x00\x00\x00\x00\x00]\x94}\x94(\x8c\x07Message\x94\x8c\x05HELLO\x94\x8c\x04Peer\x94\x8c\x02P7\x94ua.'
2023-03-25 13:50:00,868 - INFO - [RECEIVER] - Added neighbour P8
2023-03-25 13:50:59,158 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:50:59,744 - INFO - [RecvLoop_213.232] - On destroy: circuit #80000002
2023-03-25 13:50:59,744 - INFO - [RecvLoop_213.232] - Destroy circuit #80000002
2023-03-25 13:51:38,159 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:51:38,571 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:52:20,337 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:53:14,688 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:53:15,729 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:54:12,390 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:55:11,514 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:11,776 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:13,279 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:13,511 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:13,887 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:55:14,052 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:55:14,390 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:17,270 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:18,296 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:18,476 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:18,477 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:18,487 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:18,673 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:22,688 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:23,375 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:23,643 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:23,644 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:23,870 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:24,747 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:26,326 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:26,545 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:27,326 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:27,545 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:27,547 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:27,991 - INFO - [Thread-4] - Adding P5 to list of inactive
2023-03-25 13:55:27,991 - INFO - [Thread-4] - Removing P5 from neighbours
2023-03-25 13:55:28,367 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:28,613 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:31,145 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:34,089 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:39,080 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:41,798 - ERROR - [RecvLoop_97.113.] - Some handle errors
Traceback (most recent call last):
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 224, in _do_recv
    self._handler_mgr.handle(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 280, in handle
    handler(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 43, in wrapped
    return func(*args_new, **kwargs)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/guard.py", line 127, in _on_relay
    circuit.handle_relay(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 456, in handle_relay
    circuit_node, inner_cell = self._decrypt(cell)
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/circuit.py", line 522, in _decrypt
    return from_node, relay_cell.get_decrypted()
  File "/home/vboxuser/Git/imt4905/torpy-rtun-fork/torpy/cells.py", line 234, in get_decrypted
    assert self._checked
AssertionError
2023-03-25 13:55:48,138 - INFO - [CONNECTION] - Neighbour connections (1) less than MIN_NEIGHBOUR_CONNECTIONS (2)
2023-03-25 13:55:48,138 - INFO - [CONNECTION] - Candidate peers: ['P4', 'P3', 'P6', 'P2', 'P1']
2023-03-25 13:55:48,138 - INFO - [CONNECTION] - Highest cost peers: [('P2', 3431), ('P6', 3136), ('P1', 2713), ('P3', 2613), ('P8', 1743), ('P4', 1736), ('P5', 1636), ('P7', 0)]
2023-03-25 13:55:48,143 - INFO - [CONNECTION] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:55:48,531 - INFO - [CONNECTION] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:55:48,547 - INFO - [MainThread] - ['ESTABLISH', 'RunningOnFumes2', b'7a30ac6789cd9ce14f9c']
2023-03-25 13:55:48,547 - INFO - [MainThread] - Got ESTABLISH to relay RunningOnFumes2:b'7a30ac6789cd9ce14f9c' via NeelTorRelay2
2023-03-25 13:55:48,548 - INFO - [Establish-RunningOnFumes2] - Establishing RP to RunningOnFumes2
2023-03-25 13:55:48,555 - INFO - [Establish-RunningOnFumes2] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:55:48,835 - INFO - [Establish-RunningOnFumes2] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:55:48,836 - INFO - [Establish-RunningOnFumes2] - Connecting to guard node 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13)... (None)
2023-03-25 13:55:49,612 - INFO - [Establish-RunningOnFumes2] - Creating new circuit #80000003 with 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13) router...
2023-03-25 13:55:49,799 - INFO - [Establish-RunningOnFumes2] - Building -1 hops circuit...
2023-03-25 13:55:49,800 - INFO - [Establish-RunningOnFumes2] - Extending the circuit #80000003 with 217.12.221.131:443 (RunningOnFumes2; Tor 0.4.7.13)...
2023-03-25 13:55:49,800 - INFO - [Establish-RunningOnFumes2] - Getting descriptor for 217.12.221.131:443 (RunningOnFumes2; Tor 0.4.7.13)...
2023-03-25 13:55:49,805 - INFO - [Establish-RunningOnFumes2] - Connecting to guard node 185.220.101.73:9000 (CCCStuttgartBer; Tor 0.4.7.13)... (Internal dir client)
2023-03-25 13:55:50,035 - INFO - [Establish-RunningOnFumes2] - Creating new circuit #80000004 with 185.220.101.73:9000 (CCCStuttgartBer; Tor 0.4.7.13) router...
2023-03-25 13:55:50,097 - INFO - [Establish-RunningOnFumes2] - Building 0 hops circuit...
2023-03-25 13:55:50,097 - INFO - [Establish-RunningOnFumes2] - Stream #3: creating attached to #80000004 circuit...
2023-03-25 13:55:50,098 - INFO - [Establish-RunningOnFumes2] - Stream #3: connecting to hsdir
2023-03-25 13:55:50,158 - INFO - [RecvLoop_185.220] - Stream #3: connected (remote ip '')
2023-03-25 13:55:50,454 - INFO - [RecvLoop_185.220] - Stream #3: remote disconnected (reason = DONE)
2023-03-25 13:55:50,454 - INFO - [Establish-RunningOnFumes2] - Stream #3: closing (state = Disconnected)...
2023-03-25 13:55:50,454 - INFO - [Establish-RunningOnFumes2] - Got descriptor
2023-03-25 13:55:50,839 - INFO - [Establish-RunningOnFumes2] - <class 'torpy.circuit.TorCircuit'>
2023-03-25 13:55:51,254 - INFO - [Establish-RunningOnFumes2] - Rendezvous established (CellRelayRendezvousEstablished())
2023-03-25 13:55:51,261 - INFO - [Establish-RunningOnFumes2] - Waiting for connections at relay RunningOnFumes2 for cookie b'7a30ac6789cd9ce14f9c' ...
2023-03-25 13:55:58,588 - INFO - [CONNECTION] - Sending JOIN to P2 at relay RunningOnFumes2
2023-03-25 13:56:09,514 - WARNING - [Connect-Vulnerar] - Got SENDME
2023-03-25 13:56:09,902 - ERROR - [Establish-RMK] - recv timeout
2023-03-25 13:56:09,902 - INFO - [Establish-RMK] - Closing connection Establish-RMK
2023-03-25 13:56:09,902 - INFO - [Establish-RMK] - Stream #2: closing (state = Connected)...
2023-03-25 13:56:09,903 - INFO - [Establish-RMK] - Destroy circuit #80000001
2023-03-25 13:56:09,903 - INFO - [Establish-RMK] - Closing guard connections (None)...
2023-03-25 13:56:14,667 - ERROR - [RECEIVER] - Error in ReceiveThread: 'P5'
2023-03-25 13:56:29,115 - INFO - [Thread-4] - Adding P8 to list of inactive
2023-03-25 13:56:29,115 - INFO - [Thread-4] - Removing P8 from neighbours
2023-03-25 13:56:58,622 - INFO - [CONNECTION] - Neighbour connections (0) less than MIN_NEIGHBOUR_CONNECTIONS (2)
2023-03-25 13:56:58,622 - INFO - [CONNECTION] - Candidate peers: ['P3', 'P2', 'P1', 'P4']
2023-03-25 13:56:58,622 - INFO - [CONNECTION] - Highest cost peers: [('P2', 3431), ('P1', 2713), ('P3', 2613), ('P6', 1843), ('P8', 1743), ('P4', 1736), ('P5', 1636), ('P7', 0)]
2023-03-25 13:56:58,638 - INFO - [CONNECTION] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:56:58,904 - INFO - [CONNECTION] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:56:58,915 - INFO - [MainThread] - ['ESTABLISH', 'Blizzardpaw', b'92b9cbf9dc80ed67acf6']
2023-03-25 13:56:58,916 - INFO - [MainThread] - Got ESTABLISH to relay Blizzardpaw:b'92b9cbf9dc80ed67acf6' via NeelTorRelay2
2023-03-25 13:56:58,917 - INFO - [Establish-Blizzardpaw] - Establishing RP to Blizzardpaw
2023-03-25 13:56:58,926 - INFO - [Establish-Blizzardpaw] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:56:59,207 - INFO - [Establish-Blizzardpaw] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:56:59,209 - INFO - [Establish-Blizzardpaw] - Connecting to guard node 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13)... (None)
2023-03-25 13:56:59,683 - INFO - [RecvLoop_185.220] - On destroy: circuit #80000004
2023-03-25 13:56:59,683 - INFO - [RecvLoop_185.220] - Destroy circuit #80000004
2023-03-25 13:56:59,980 - INFO - [Establish-Blizzardpaw] - Creating new circuit #80000005 with 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13) router...
2023-03-25 13:57:00,170 - INFO - [Establish-Blizzardpaw] - Building -1 hops circuit...
2023-03-25 13:57:00,170 - INFO - [Establish-Blizzardpaw] - Extending the circuit #80000005 with 185.7.33.146:443 (Blizzardpaw; Tor 0.4.7.13)...
2023-03-25 13:57:00,171 - INFO - [Establish-Blizzardpaw] - Getting descriptor for 185.7.33.146:443 (Blizzardpaw; Tor 0.4.7.13)...
2023-03-25 13:57:00,178 - INFO - [Establish-Blizzardpaw] - Connecting to guard node 212.159.69.155:443 (FuckYouPutin; Tor 0.4.7.8)... (Internal dir client)
2023-03-25 13:57:00,402 - INFO - [Establish-Blizzardpaw] - Creating new circuit #80000006 with 212.159.69.155:443 (FuckYouPutin; Tor 0.4.7.8) router...
2023-03-25 13:57:00,474 - INFO - [Establish-Blizzardpaw] - Building 0 hops circuit...
2023-03-25 13:57:00,475 - INFO - [Establish-Blizzardpaw] - Stream #4: creating attached to #80000006 circuit...
2023-03-25 13:57:00,475 - INFO - [Establish-Blizzardpaw] - Stream #4: connecting to hsdir
2023-03-25 13:57:00,527 - INFO - [RecvLoop_212.159] - Stream #4: connected (remote ip '')
2023-03-25 13:57:00,807 - INFO - [RecvLoop_212.159] - Stream #4: remote disconnected (reason = DONE)
2023-03-25 13:57:00,809 - INFO - [Establish-Blizzardpaw] - Stream #4: closing (state = Disconnected)...
2023-03-25 13:57:00,810 - INFO - [Establish-Blizzardpaw] - Got descriptor
2023-03-25 13:57:01,182 - INFO - [Establish-Blizzardpaw] - <class 'torpy.circuit.TorCircuit'>
2023-03-25 13:57:01,555 - INFO - [Establish-Blizzardpaw] - Rendezvous established (CellRelayRendezvousEstablished())
2023-03-25 13:57:01,555 - INFO - [Establish-Blizzardpaw] - Waiting for connections at relay Blizzardpaw for cookie b'92b9cbf9dc80ed67acf6' ...
2023-03-25 13:57:09,202 - INFO - [CONNECTION] - Sending JOIN to P2 at relay Blizzardpaw
2023-03-25 13:58:06,498 - INFO - [RecvLoop_212.159] - On destroy: circuit #80000006
2023-03-25 13:58:06,499 - INFO - [RecvLoop_212.159] - Destroy circuit #80000006
2023-03-25 13:58:09,215 - INFO - [CONNECTION] - Neighbour connections (0) less than MIN_NEIGHBOUR_CONNECTIONS (2)
2023-03-25 13:58:09,215 - INFO - [CONNECTION] - Candidate peers: ['P3', 'P2', 'P1', 'P4']
2023-03-25 13:58:09,215 - INFO - [CONNECTION] - Highest cost peers: [('P2', 3431), ('P1', 2713), ('P3', 2613), ('P6', 1843), ('P8', 1743), ('P4', 1736), ('P5', 1636), ('P7', 0)]
2023-03-25 13:58:09,243 - INFO - [CONNECTION] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:58:09,646 - INFO - [CONNECTION] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:58:09,659 - INFO - [MainThread] - ['ESTABLISH', 'apollon', b'38f2e45ed1e749a83cec']
2023-03-25 13:58:09,659 - INFO - [MainThread] - Got ESTABLISH to relay apollon:b'38f2e45ed1e749a83cec' via NeelTorRelay2
2023-03-25 13:58:09,660 - INFO - [Establish-apollon] - Establishing RP to apollon
2023-03-25 13:58:09,669 - INFO - [Establish-apollon] - Loading cached NetworkStatusDocument from TorCacheDirStorage: /root/.local/share/torpy/network_status
2023-03-25 13:58:09,947 - INFO - [Establish-apollon] - Loading cached DirKeyCertificateList from TorCacheDirStorage: /root/.local/share/torpy/dir_key_certificates
2023-03-25 13:58:09,949 - INFO - [Establish-apollon] - Connecting to guard node 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13)... (None)
2023-03-25 13:58:10,784 - INFO - [Establish-apollon] - Creating new circuit #80000007 with 97.113.253.84:110 (NeelTorRelay2; Tor 0.4.7.13) router...
2023-03-25 13:58:10,972 - INFO - [Establish-apollon] - Building -1 hops circuit...
2023-03-25 13:58:10,972 - INFO - [Establish-apollon] - Extending the circuit #80000007 with 212.147.124.158:9002 (apollon; Tor 0.4.7.13)...
2023-03-25 13:58:10,978 - INFO - [Establish-apollon] - Getting descriptor for 212.147.124.158:9002 (apollon; Tor 0.4.7.13)...
2023-03-25 13:58:10,984 - INFO - [Establish-apollon] - Connecting to guard node 85.215.88.244:9001 (BlackMesaAG; Tor 0.4.7.13)... (Internal dir client)
2023-03-25 13:58:11,231 - INFO - [Establish-apollon] - Creating new circuit #80000008 with 85.215.88.244:9001 (BlackMesaAG; Tor 0.4.7.13) router...
2023-03-25 13:58:11,283 - INFO - [Establish-apollon] - Building 0 hops circuit...
2023-03-25 13:58:11,284 - INFO - [Establish-apollon] - Stream #5: creating attached to #80000008 circuit...
2023-03-25 13:58:11,284 - INFO - [Establish-apollon] - Stream #5: connecting to hsdir
2023-03-25 13:58:11,347 - INFO - [RecvLoop_85.215.] - Stream #5: connected (remote ip '')
2023-03-25 13:58:11,550 - INFO - [RecvLoop_85.215.] - Stream #5: remote disconnected (reason = DONE)
2023-03-25 13:58:11,551 - INFO - [Establish-apollon] - Stream #5: closing (state = Disconnected)...
2023-03-25 13:58:11,551 - INFO - [Establish-apollon] - Got descriptor
2023-03-25 13:58:11,975 - INFO - [Establish-apollon] - <class 'torpy.circuit.TorCircuit'>
2023-03-25 13:58:12,353 - INFO - [Establish-apollon] - Rendezvous established (CellRelayRendezvousEstablished())
2023-03-25 13:58:12,354 - INFO - [Establish-apollon] - Waiting for connections at relay apollon for cookie b'38f2e45ed1e749a83cec' ...
2023-03-25 13:58:19,970 - INFO - [CONNECTION] - Sending JOIN to P2 at relay apollon
